app/Controllers/LoanApprovalController.php

```
@@ -234,14 +236,24 @@ public function approveLoan(Request $request, Response $response, mysqli $db): R
-            } catch (\Throwable $e) {
-                $today = date('Y-m-d');
-            }
+            // DateHelper::today() incapsula già i fallback sul timezone (M9).
+            $today = DateHelper::today();
 
             $utenteId = (int) $loan['utente_id'];
+
+            // M7 — gate di idoneità anche in APPROVAZIONE: l'utente può essere
+            // stato sospeso (o la tessera può essere scaduta) tra la richiesta e
+            // l'approvazione admin. Stesso gate usato da store()/createReservation.
+            $eligibilityError = \App\Support\LoanEligibility::checkUser($db, $utenteId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\LoanEligibility' in method 'approveLoan'. (undefined)

(StaticAccess)